### PR TITLE
feat(website): add Chainstack provider in 'Run a Taiko node'

### PR DESCRIPTION
**Pull request by soos3d**
Hi, I added the instructions on using Chainstack to get an RPC to run a Taiko node (in the same section where Infura and Alchemy are mentioned). 

Full disclosure: I'm a dev advocate at Chainstack.

### CODE REVIEW

**Comment by samui68400** [comment]
I review it and I find it perfectly clear, bravo

**Review by 2manslkh — APPROVED** [review]

File: packages/website/package-lock.json
Comment (2manslkh) [inline packages/website/package-lock.json:1]: Please uncommit this file

**Review by 2manslkh — CHANGES_REQUESTED** [review]
remove the package-lock.json file

**Comment by dantaik** [comment]
Please notice that the website code has been migrated to https://github.com/taikoxyz/website, please submit PRs to the new repository. Thank you.

**Pull request closed without merging**